common/src/models/domain/encString.ts

```diff
@@ -96,11 +96,10 @@ export class EncString {
     }
   }
 
+  /**
+   * @deprecated As of 2022-04-20, use {@link decryptWithCryptoService} instead
+   */
   async decrypt(orgId: string, key: SymmetricCryptoKey = null): Promise<string> {
-    if (this.decryptedValue != null) {
-      return this.decryptedValue;
-    }
-
```

Comment:
I don't think it's a good idea to remove this guard.

Are you thinking that the object hierarchy makes multiple decrypts less likely?

Reply:
It's called in `decryptWithCryptoService`. So the guard is still there just moved below the container service initialization.